misc/ninja_syntax.py

```diff
@@ -21,8 +21,11 @@ def __init__(self, output, width=78):
     def newline(self):
         self.output.write('\n')
 
-    def comment(self, text):
-        for line in textwrap.wrap(text, self.width - 2):
+    def comment(self, text, has_path=False):
+        args = {}
+        if has_path:
```

Comment:
I think I agree with Nico -- it's fine if a comment with a very long word goes over the width, because the only reason it would is if it has a long word which is unlikely to be splittable.